src/pages/Index/Aside/Recents/Watchlists.js

```diff
@@ -23,22 +24,29 @@ const Watchlist = ({ name, isPublic, historicalStats }) => {
   if (!name) return null
 
   const lastData = historicalStats[historicalStats.length - 1]
-  const marketcap = lastData ? lastData.marketcap : null
+  const firstData = historicalStats[0]
+  const marketcapLast = lastData ? lastData.marketcap : 0
+  const marketcapFirst = firstData ? firstData.marketcap : 0
+
+  const change = (marketcapLast - marketcapFirst) / marketcapFirst
```

Comment:
in case when we don't have any data in `historicalStats` (empty watchlist / screener) - we will get `NaN`